platform/cog-platform-fdo.c

```diff
@@ -340,6 +355,68 @@ static const struct zxdg_toplevel_v6_listener xdg_toplevel_listener = {
     .close = xdg_toplevel_on_close,
 };
 
+
+static void noop()
```

Comment:
Please put the function type in its own line, with a space between the name and the opening paren for the parameter list. Also, I think that the comment inside the function is not needed because the name “noop” already implies that it does nothing: 

```c
static void
noop (void)
{
}
```